crates/bevy_ecs/src/reflect.rs

```diff
@@ -99,10 +99,6 @@ impl ReflectComponent {
     }
 
     /// Removes this [`Component`] type from the entity. Does nothing if it doesn't exist.
-    ///
-    /// # Panics
-    ///
-    /// Panics if there is no [`Component`] of the given type.
```

Comment:
So `entity.remove::<T>()` won't panic if the component doesn't exist on the entity? (Did it even panic before?)

Reply:
Gotcha. Well `ReflectComponent` only operates on a single component anyways, so either should work fine 😄